formats/utils/mime.ts

```diff
@@ -996,3 +997,7 @@ export function extension(type: string) {
 
 	return ext || '';
 }
+
+export function mime(extension: string) {
```

Comment:
I'm seeing that this is only used to convert an extension to a mime and then immediately converted back to an extension. Why not just use the extension?

Reply:
It is removed from the PR now because it is no longer needed.

Reply:
Also, speaking of `mime.ts`, the data seems inadequate. For example, `image/jpeg` only maps to `.jpeg` but not `.jpg`.